### PR TITLE
Editing README

### PR DESCRIPTION
**Pull request by jessethach**
Adding location of mock data in README. Adding filtering options in README. Adding mock data json in mock data dir.

### CODE REVIEW

File: src/app/shared/mock-data/mock-data.md [hunk 1]
```diff
@@ -3,3 +3,125 @@
 This directory contains the data that you may use to mock API data with.
 
 **Hint** You may want to create a model for this data.
+
```
Comment (itrelouis) [inline src/app/shared/mock-data/mock-data.md]: is this just duplicating cars-data.ts? seems redundant

Comment (jessethach) [reply src/app/shared/mock-data/mock-data.md]: Definitely is - didn't want them to somehow miss it ¯\_(ツ)_/¯ I can remove it.

Comment (itrelouis) [reply src/app/shared/mock-data/mock-data.md]: maybe just link to the data files?  My concern would be that they have to change then you have to remember to change them here too

Comment (emarzinskyiTrellis) [reply src/app/shared/mock-data/mock-data.md]: I would remove one of the mock data files. Seems fine to just have the cars one. They can always add another file if they need it later. There's already a good amount of hand-holding in this project.

Comment (itrelouis) [reply src/app/shared/mock-data/mock-data.md]: ugh clicked the review button by accident

Comment (jessethach) [reply src/app/shared/mock-data/mock-data.md]: Ok, I'll leave the link to the mock data file in the root README and remove the data in this README file.

Comment (jessethach) [reply src/app/shared/mock-data/mock-data.md]: Also link it in the directory level

**Pull request merged**